front/src/front/static/js/script.js

```diff
@@ -0,0 +1,83 @@
+let isDarkMode = false;
+
+const HOST = window.location.protocol + '//'
+    + window.location.host + '/';
+
+const navbarBrandElement = document.querySelector('.navbar-brand');
+navbarBrandElement.addEventListener('click', homeNav);
+history.pushState({'previousComponent': '/'}, null, HOST);
+
+const logPart = document.querySelector('#log-part');
+const logoutPart = document.querySelector('#logout-part')
+const navUsername = document.querySelector('#nav-username');
+const navProfileImg = document.querySelector('#nav-profile-img');
+
+function switchMode() {
+    if (!isDarkMode) {
+        document.querySelector('body').setAttribute('data-bs-theme', 'dark');
+        document.querySelector('#switch-btn').classList.remove('btn-outline-dark');
+        document.querySelector('#switch-btn').setAttribute('class', 'btn btn-outline-light')
+        isDarkMode = true;
+    } else {
+        document.querySelector('body').setAttribute('data-bs-theme', 'light');
+        document.querySelector('#switch-btn').classList.remove('btn-outline-light');
+        document.querySelector('#switch-btn').setAttribute('class', 'btn btn-outline-dark');
+        isDarkMode = false;
+    }
+}
```

Comment:
You can declare
`document.querySelector('body')`
and
`document.querySelector('#switch-btn')`
as const to avoid repetitive function call

Reply:
https://github.com/tdameros/42-transcendence/pull/23/commits/c4085fd6c3757fded8f1757d0e593784b05823f1